README.md

```diff
@@ -66,15 +66,20 @@ The [Lighthouse](https://github.com/submariner-io/lighthouse) project helps in c
 
 To deploy Submariner with Lighthouse follow the below steps.
 
+* Create a kubeconfig file which has the context of all the clusters. If KUBECONFIG environment variable set with the path to all clusters, you could use the below command.
+```
+kubectl config view --flatten > kubeconfig_merged
```

Comment:
After creating "kubeconfig_merged", where should it be used ?
Please specify command.

Reply:
I have added the details in the join command, where this should be used.

Reply:
I have added more details on the README based on your feedback.